src/codewhisperer/service/transformationHubViewProvider.ts

```diff
@@ -237,9 +250,9 @@ export class TransformationHubViewProvider implements vscode.WebviewViewProvider
         if (stepStatus === StepProgress.Succeeded) {
             return `<span style="color: green"> ✓ </span>`
         } else if (stepStatus === StepProgress.Pending) {
-            return `<span> <i class="fas fa-spinner fa-spin"></i> </span>` // TODO: switch from FA to native VSCode icons
+            return `<span style="color: grey" class="spinner"> ↻ </span>`
         } else {
-            return `<span style="color: red"> X </span>`
+            return `<span style="color: grey"> ✓ </span>`
```

Comment:
Our PM and UX team decided that they want to display a grey checkmark rather than a red X when a transformation step fails since the transformation was in-progress / partially completed.